### PR TITLE
feat: allow to override admin and viewer token keys in secret for adm…

### PR DESCRIPTION
**Pull request by ikatlinsky**
Provides a way to override names of secret keys for admin API credentials, as per proposal https://github.com/apache/apisix-helm-chart/issues/550.

### CODE REVIEW

**Comment by ikatlinsky** [comment]
@tao12345666333 @tokers can i ask you to take a look at the proposed changes and share your ideas?

**Review by lingsamuel — APPROVED** [review]

File: charts/apisix/templates/_helpers.tpl [hunk 1]
```diff
@@ -111,3 +111,25 @@ Scheme to use while connecting etcd
...
+{{/*
+Key to use to fetch viewer token from secret
+*/}}
+{{- define "apisix.admin.credentials.secretViewerKey" -}}
+{{- if .Values.admin.credentials.secretViewerKey }}
+{{- .Values.admin.credentials.secretViewerKey }}
+{{- else }}
+{{- "viewer" }}
+{{- end }}
+{{- end }}
```
Comment (Gallardot) [inline charts/apisix/templates/_helpers.tpl:135]: need new line at end of file

**Review by Gallardot — COMMENTED** [review]
Rest LGTM

**Review by Gallardot — APPROVED** [review]

**Review by tao12345666333 — COMMENTED** [review]
Sorry for the delay. 

I love your method. Could you please resolve the conflicts, let's merge this PR. Thanks!

**Comment by ikatlinsky** [comment]
Done, no problem, glad to help 🤝

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Pull request merged**